src/deep_links/mod.rs

```diff
@@ -39,14 +39,15 @@ pub struct OpenPlayerLink {
 #[derive(Default, Serialize, Debug, PartialEq, Eq)]
 #[serde(rename_all = "camelCase")]
 pub struct ExternalPlayerLink {
-    pub href: Option<String>,
     pub download: Option<String>,
     pub streaming: Option<String>,
+    pub playlist: Option<String>,
+    pub file_name: Option<String>,
     pub open_player: Option<OpenPlayerLink>,
-    pub android_tv: Option<String>,
+    pub web: Option<Url>,
```

Comment:
Please write a descriptive comment on the field, e.g.:
```suggestion
   /// A Url used for an external player
    pub web: Option<Url>,
```